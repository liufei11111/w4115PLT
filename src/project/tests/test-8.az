Int main(Int argc, String argv)｛
Int asdf;
Int ASDF;
Int A0a9;
Int asdf____;
Int asdf_ASDF;
｝