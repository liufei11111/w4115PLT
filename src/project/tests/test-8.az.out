
public static int main(int argc, String argv)
{
try{int asdf=0;
int ASDF=0;
int A0a9=0;
int asdf____=0;
int asdf_ASDF=0;
}catch(Exception e){return -1;}}

